src/OpcApplicationConfiguration.cs

```diff
@@ -75,117 +77,117 @@ public OpcApplicationConfiguration()
         /// </summary>
         public async Task<ApplicationConfiguration> ConfigureAsync()
         {
-            // instead of using a configuration XML file, we configure everything programmatically
-
-            // passed in as command line argument
-            ApplicationConfiguration = new ApplicationConfiguration
-            {
+            // instead of using a configuration XML file, configure everything programmatically
+            var application = new ApplicationInstance() {

```

Comment:
We use curly braces in the next line for this solution. Please reformat document.